src/layouts/Lightwell.scss

```
@@ -0,0 +1,24 @@
+@media (min-width: $pf-v6-global--breakpoint--xl) {
+  .pf-v6-c-page.chr-c-page--lightwell {
+    grid-template-areas: 'header' 'subnav' 'main';
+    grid-template-columns: 1fr;
+    grid-template-rows: auto auto 1fr;
+    --pf-v6-c-page--masthead--main-container--GridArea: main;
+    --pf-v6-c-page__main-container--GridArea: main;
+
+    > .pf-v6-c-masthead {
+      // Prevent subgrid — keep inline masthead layout
+      --pf-v6-c-masthead--m-display-inline--GridTemplateColumns: min-content 1fr;
+      grid-template-columns: min-content 1fr;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)